frontend/src/components/pages/CampCreation/ScheduleSessions/SidePanel/ScheduledSessionsCard.tsx

```diff
@@ -14,7 +14,7 @@ const ScheduledSessionsCard = ({
   currIndex,
   scheduledSession,
   onDelete,
-}: ScheduledSessionsCardProps): JSX.Element => {
+}: ScheduledSessionsCardProps): React.ReactElement => {
```

Comment:
is this branch rebased? I tried deleting sessions from the current sessions panel but it didn't show the delete session modal to confirm delete. I thought Ryan already changed that on his end?
<img width="498" alt="image" src="https://user-images.githubusercontent.com/57159970/202932837-6fd87837-ee4c-41a7-a3b5-f59a5d7d177e.png">

Reply:
Hadn't rebased, will be merged in https://github.com/uwblueprint/focus-on-nature/pull/161